docs/blockchain-oef-usage.md

```diff
@@ -0,0 +1,54 @@
+The AEA framework enables the agents to interact with the blockchain in the form of transaction completion. Currently, the framework supports
+two different networks natively: the `Fetch.ai` network and the `Ethereum` network.
+
+Consider the following use case:
+
+Once the AEAs successfully end the negotiations, the buyer should send the correct amount to the specified wallet address. Then, the provider
+validates the transaction digest and sends the data to the client.
+
+The limitation of the previous use case is that we should trust the provider that he will send the data upon successful payment.
+
+## Ledger Apis
+
+We interact with the blockchains through a module called `LedgerApis`. The `ledger_apis` module contains implementations of each ledger API object (abstract class) that wraps the libraries of each network. With these implementations, we can create accounts for the specified networks. The account contains the private/public key pair and a wallet address. Also with the account, we will be able to
```

Comment:
The `LedgerApis` module

Reply:
of each network, in particular `FetchAILedgerApi` and `EthereumLedgerApi`.

Reply:
Accounts are not created in the ledger apis..these are created in the crypto modules.